pkg/collector/reconcile/configmap.go

```diff
@@ -117,10 +117,19 @@ func expectedConfigMaps(ctx context.Context, params Params, expected []corev1.Co
 			updated.ObjectMeta.Labels[k] = v
 		}
 
-		if err := params.Client.Update(ctx, updated); err != nil {
+		// editting Update with Patch
+		patch := client.MergeFrom(&params.Instance)
+
+		// if err := params.Client.Patch(ctx, &changed, patch); err != nil {
```

Comment:
You can also remove the commented out code. If we need it later, we can browse the git history.